pkg/server/sandbox_run.go

```diff
@@ -69,7 +69,8 @@ func (c *criService) setupPod(ctx context.Context, id string, path string, confi
 	}
 
 	labels := getPodCNILabels(id, config)
-	result, err := c.netPlugin.Setup(id,
+	result, err := c.netPlugin.Setup(ctx,
```

Comment:
did the `netPlugin` interface change when we upgraded to a new version of containerd?

Reply:
It was updated in go-cni. We could force use a previous version of the package if preferred.

Reply:
no preference, just curious why it changed.